tools/cli/pkg/command/upgrade_cluster.go

```diff
@@ -44,7 +44,9 @@ func (cmd *UpgradeClusterCommand) Validate() error {
 	if err != nil {
 		return err
 	}
-
+	if GlobalOpts.SlackAPIURL() == "" {
```

Comment:
Should it be required? 
I would say let's make this optional, and if it's unset, just ignore sending slack message

Reply:
Hi @ebensom , now make it as optional  and only print not `sending notification when its empty` .